app/models/ems_event.rb

```diff
@@ -27,7 +27,7 @@ def self.task_final_events
   def self.event_groups
     core_event_groups = ::Settings.event_handling.event_groups.to_hash
     Settings.ems.each_with_object(core_event_groups) do |(_provider_type, provider_settings), event_groups|
-      provider_event_groups = provider_settings.fetch_path(:event_handling, :event_groups)
+      provider_event_groups = provider_settings.try(:fetch_path, :event_handling, :event_groups)
```

Comment:
@jrafanie if all behave correctly, its always a hash. But see the link in the description. Somebody could break it by setting something, which is not a hash.